keps/sig-node/4960-container-stop-signals/kep.yaml

```diff
@@ -0,0 +1,31 @@
+title: Container Stop Signals
+kep-number: "4960"
+authors: 
+  - '@sreeram-venkitesh'
+owning-sig: sig-node
+participating-sigs: 
+  - sig-node
```

Comment:
probably sig-windows as it is mentioned in comments.

Reply:
Added in https://github.com/kubernetes/enhancements/pull/5122/commits/9508d34147087bc0bca8967280ba2c875635e700